Sources_v3/Database/DTOs/UserDTO.swift

```diff
@@ -33,7 +33,7 @@ extension UserDTO {
     ///   - id: The id of the user to fetch
     ///   - context: The context used to fetch `UserDTO`
     ///
-    fileprivate static func load(id: String, context: NSManagedObjectContext) -> UserDTO? {
+    static func load(id: String, context: NSManagedObjectContext) -> UserDTO? {
```

Comment:
Why this change? Isn't `context.user(id:)` what you need?

Reply:
```swift
func user(id: UserId) -> UserDTO? {
        UserDTO.load(id: id, context: self)
    }
```
I need `loadOrCreate`

Reply:
Ooh you mean I didn't need `UserDTO.load`.. right, i'll restore `fileprivate`

Reply:
Fixed in 62b46eb0b0bef360c33b591d1dbceb93446701e2